### PR TITLE
 Fix #12: Check to retrieve last approvals of each collection

### LINKED ISSUE #12: Add check that retrieves the last X approval datetimes + number of changes

### PR DESCRIPTION
**Pull request by leplatrem**
Based on #21 

Requires https://github.com/Kinto/kinto-http.py/pull/214

* [x] Basic feature
* [x] Add tests
* [x] See how to retrieve number of changes

### CODE REVIEW

File: checks/remotesettings/last_approvals.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+
+def get_last_approvals(client, bucket, collection, max_approvals):
+    history = client.get_history(
+        bucket=bucket,
+        **{
+            "resource_name": "collection",
+            "target.data.id": collection,
+            "target.data.status": "to-sign",
+            "_limit": max_approvals,
```
Comment (glasserc) [inline checks/remotesettings/last_approvals.py]: Should we provide a `_sort`?

File: checks/remotesettings/last_approvals.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        }
+    )
+
+    return [{"date": h["date"], "by": h["user_id"]} for h in history]
```
Comment (glasserc) [inline checks/remotesettings/last_approvals.py]: In Zoom you mentioned getting the number of approved changes too.

File: checks/remotesettings/last_approvals.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+"""
+Dummy-check to obtain dates of last approvals by collection.
```
[the remark this answers is not in the document]
Comment (leplatrem) [reply checks/remotesettings/last_approvals.py]: I rephrased the module docstring. Please let me know if it's clearer :/

**Comment by leplatrem** [comment]
I get your point! 

I think we could cover both needs just by sorting the entries by the latest approval datetime descending. Let me know how do you feel about that.

Limiting the number of entries was mainly to limit the response size and keeping it human-readable.

**Comment by glasserc** [comment]
> I think we could cover both needs just by sorting the entries by the latest approval datetime descending.

I think sorting by approval means it's more in the "latest" category (i.e. not exhaustive) and I think we should rename it accordingly. But I don't feel strongly about it and I won't withdraw my approval.

**Comment by leplatrem** [comment]
It makes total sense :+1:

**Pull request merged**